src/cpp/shared_core/FileLogDestination.cpp

```diff
@@ -574,6 +574,12 @@ void FileLogDestination::writeLog(LogLevel in_logLevel, const std::string& in_me
       if (!m_impl->openLogFile())
       {
          m_impl->closeLogFile();
+
+#ifndef _WIN32
```

Comment:
Perhaps instead of duplicating this here, we just move the original syslog write call to the beginning of the `writeLog` method. Then we log to file after writing to syslog.

Reply:
Yeah, great idea.  I'll make that change.